bytecheck/Cargo.toml

```diff
@@ -14,7 +14,7 @@ readme = "crates-io.md"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-bytecheck_derive = { version = "=0.6.9", path = "../bytecheck_derive", default-features = false }
+bytecheck_derive = { version = "=0.7.0", path = "../bytecheck_derive", default-features = false }
```

Comment:
should this be workspace versioned for the workspace crates in this repo? 

```toml
[package]
name = "bytecheck"
version.workspace = true
edition.workspace = true
```

Reply:
Probably yeah. If you want, a follow-up PR would be ideal.